sql/load_demo_data.sql

```diff
@@ -6537,7 +6760,11 @@ CREATE TABLE `oh_ward` (
   `WRD_LAST_MODIFIED_BY` varchar(50) DEFAULT NULL,
```

Comment:
The column WRD_ID_A doesn't have a default value in the demo data version.

Reply:
It is correct, it should be NOT NULL and without a default data.
I think to address the change in OP-1107 with a new step_XX. I will report under the issue.
Thanks!